packages/control-property-editor/src/App.tsx

```diff
@@ -130,26 +133,26 @@ export default function App(appProps: AppProps): ReactElement {
                 <section className="app-panel app-panel-right">
                     <PropertiesList />
                 </section>
-                {isAdpProject && shouldHideIframe && (
-                    <UIDialog
-                        hidden={!shouldShowDialogMessage}
-                        dialogContentProps={{
-                            title: t('TOOL_DISCLAIMER_TITLE'),
-                            subText: dialogMessage?.message
-                        }}
-                    />
-                )}
-                {isAdpProject && !shouldHideIframe && (
-                    <UIDialog
-                        hidden={!shouldShowDialogMessage}
-                        dialogContentProps={{
-                            title: t('TOOL_DISCLAIMER_TITLE'),
-                            subText: dialogMessage?.message
-                        }}
-                        acceptButtonText={t('OK')}
-                        onAccept={closeAdpWarningDialog}
-                    />
-                )}
+                 {isAdpProject && shouldHideIframe && dialogQueue.length > 0 && (
+                <UIDialog
```

Comment:
Please fix the formatting here, the indentation looks off.

Reply:
fixed in #[4736607](https://github.com/SAP/open-ux-tools/pull/1941/commits/47366076b46a6f40848ca3b107294662da49a582)